pages/blog/_slug.vue

```diff
@@ -36,3 +36,15 @@ export default {
   },
 };
 </script>
+
+<style>
+.content {
```

Comment:
It looks like the description underneath the author has the `content` class, so it looks funky now. Would it be easy to just change the class name for that div?

Reply:
i think it makes more sense to leave the content class for the blog text, and remove the class from the author card?